exchange/exchange.go

```diff
@@ -447,11 +459,13 @@ func applyCategoryMapping(ctx context.Context, requestExt openrtb_ext.ExtRequest
 					if dupe.bidderName == bidderName {
 						// An older bid from the current bidder
 						bidsToRemove = append(bidsToRemove, dupe.bidIndex)
+						rejections = updateRejections(rejections, dupe.bidID, "Bid was deduplicated")
 					} else {
 						// An older bid from a different seatBid we've already finished with
 						oldSeatBid := (seatBids)[dupe.bidderName]
 						if len(oldSeatBid.bids) == 1 {
 							seatBidsToRemove = append(seatBidsToRemove, bidderName)
+							rejections = updateRejections(rejections, bidID, "Bid was deduplicated")
```

Comment:
dupe.bidID should also be used here as this is still a case where we are removing the bid that was stored in dupe instead of the current one (bidID).

Reply:
Updated to use the correct duplicate bid ID